tests/azure_acr_test.go

```diff
@@ -0,0 +1,31 @@
+package test
```

Comment:
Is this redundant file.
I see the correct one at tests/terratest/azure_acr_test.go